ssl/t1_lib.c

```diff
@@ -992,13 +992,22 @@ static const uint16_t tls12_sigalgs[] = {
     TLSEXT_SIGALG_rsa_pkcs1_sha384,
     TLSEXT_SIGALG_rsa_pkcs1_sha512,
 
+/* Presense of OPENSSL_CAPIENG_LIMIT_SIGALGS removes sha-224 signature
+   algorithms. Windows does not have sha-224 implementations.
+   Pass '-DOPENSSL_CAPIENG_LIMIT_SIGALGS' to Configure. */
+#   ifndef OPENSSL_CAPIENG_LIMIT_SIGALGS
```

Comment:
Do we just allow defines like this, or do they have to be proper config options?

Reply:
We do allow defines like this.